hs_tracking/management/commands/tracking_popular.py

```diff
@@ -0,0 +1,33 @@
+"""
+Check tracking functions for proper output.
+"""
+from django.core.management.base import BaseCommand
+from hs_tracking.models import Variable
+
+
+class Command(BaseCommand):
+    help = "check on tracking"
+
+    def add_arguments(self, parser):
+        parser.add_argument('--days', type=int, dest='days', default=31,
+                            help='number of days to list')
+        parser.add_argument('--resources', type=int, dest='n_resources', default=5,
+                            help='number of resources to return')
+
+    def handle(self, *args, **options):
+        days = options['days']
+        n_resources = options['n_resources']
+
+        popular = Variable.popular_resources(days=days, n_resources=n_resources)
+        for v in popular:
+            print("users={} short_id={}"
+                  .format(v.users, v.short_id))
+            print("  title={}".format(v.title))
+            print("  created={} updated={}"
+                  .format(v.created.strftime("%Y-%m-%d %H:%M:%S"),
+                          v.last_updated.strftime("%Y-%m-%d %H:%M:%S")))
```

Comment:
hopefully we don't need to consider timezones

Reply:
This will be considered naive because I have not specified a timezone in strftime. It will be done correctly but will warn that it is using naive processing. We are not localizing timezones.